pyslurm/pyslurm.pyx

```diff
@@ -5561,9 +5574,14 @@ cdef class slurmdb_jobs:
 
     def __cinit__(self):
         self.job_cond = <slurm.slurmdb_job_cond_t *>slurm.xmalloc(sizeof(slurm.slurmdb_job_cond_t))
+        self.db_conn = slurm.slurmdb_connection_get()
 
     def __dealloc__(self):
-        pass
+        self.__free()
```

Comment:
Since `__free()` is "private" and not used elsewhere, I think we can move the `xfree` and `slurmdb_connection_close` directly under `__dealloc__`.

Reply:
no problem. i intended to follow the structure of some of the other classes.

Reply:
done ;)